mw-webapp/src/dataAccessLogic/DTOToBusinessConverter/mentorCommentDTOToMentorCommentConverter.ts

```diff
@@ -0,0 +1,32 @@
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {User} from "src/model/businessModel/User";
+import {MentorCommentDTO} from "src/model/firebaseCollection/MentorCommentDTO";
+import {UserDTO} from "src/model/firebaseCollection/UserDTO";
+
+export const MentorCommentDTOToMentorCommentConverter = (mentorComment: MentorCommentDTO, usersDTO: UserDTO[]) => {
```

Comment:
looks like wrong approach, too much calculations inside. Do you really need to get all users for just one comment?

Reply:
Its even worse in next steps, i need to transfer all WaysDTO and assign them to User, i don't understand why we need User model instead of userUuid, it would be way easier